Ports/.hosted_defs.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-SCRIPT="$(dirname "${0}")"
+SCRIPT="$(realpath $(dirname "${0}"))"
```

Comment:
The `${0}` here should be replaced with `${BASH_SOURCE[0]}`, as `${0}` will refer to the file that sourced this file instead of the sourced file itself. Otherwise, we end up sourcing the correct file, but it ends up getting the wrong value for `SCRIPT` (as it will be using `../.port_include.sh` in the wrong working directory) and therefore ends up mangling `PATH`.